apps/vessel-history/cloudbuild.yaml

```diff
@@ -93,6 +93,21 @@ steps:
         BASIC_AUTH=$_BASIC_AUTH \
         --allow-unauthenticated
 
+  - id: 'i18n-labels publish stable'
+    name: gcr.io/cloud-builders/gcloud
+    waitFor: ['deploy-cloud-run']
+    entrypoint: bash
+    env:
+      - 'WORKSPACE_ENV=$_NEXT_PUBLIC_WORKSPACE_ENV'
+    args:
+      - '-eEuo'
+      - 'pipefail'
+      - '-c'
+      - |-
+        if [ $WORKSPACE_ENV = 'production' ]; then
+          gcloud -q beta builds triggers run --branch=develop ui-libs-i18n-labels-stable
```

Comment:
shouldn't this branch be master?

Reply:
not at this stage because our most updated and stable(ish) branch is develop, however that's something we should probably revisit (do we still need master and develop?)